### ISSUE TITLE
syntax.php on line 314 - PHP 7

### ISSUE THREAD

**Issue by throny** [maphy-psd/dokuwiki-plugin-date#3]
Keep getting this error:

```
Warning: Declaration of syntax_plugin_date::handle($match, $state, $pos, &$handler) should be compatible with DokuWiki_Syntax_Plugin::handle($match, $state, $pos, Doku_Handler $handler) in D:\Apache\htdocs\wiki\lib\plugins\date\syntax.php on line 314

Warning: Declaration of syntax_plugin_date::render($mode, &$renderer, $data) should be compatible with DokuWiki_Syntax_Plugin::render($format, Doku_Renderer $renderer, $data) in D:\Apache\htdocs\wiki\lib\plugins\date\syntax.php on line 314
```

**Comment by maphy-psd** [comment]
First check: There is a pull request for that problem #1 

I will check this pull request and merge it on the weekend.

**Comment by maphy-psd** [comment]
fix by 3599cfd - close
